code/drasil-docLang/lib/Drasil/Sections/Requirements.hs

```diff
@@ -100,6 +100,17 @@ nfReqIntro :: [Contents] -> Contents
 nfReqIntro [] = mkParagraph $ emptySectSentPlu [nonfunctionalRequirement]
 nfReqIntro _  = mkParagraph $ reqIntroStart +:+. nfrReqIntroBody
 
+-- | Common Non-Functional Requirement for Maintainability.
+-- Takes in a Reference Address ('String'), a percent value ('Integer'), 
+-- and a label ('String').
+mkMaintainableNFR :: String -> Integer -> String -> ConceptInstance
+mkMaintainableNFR refAddress percent lbl = cic refAddress (foldlSent [
+  S "If a likely change is made" `S.toThe` 
+  S "finished software, it will take", addPercent percent `S.ofThe`
```

Comment:
What do you think of "it will take _at most_ X%" instead of an exact %?

Reply:
I think we should make the change suggested by @balacij.  The 10% is a the bound on the time.  The developer can do the redevelopment (change) in less than 10% of the time.